app/models/base.rb

```diff
@@ -31,4 +32,28 @@ def app_and_serv_check(appliance_id, service_id, appdb_data)
 
     raise Errors::NotFoundError, "Appliance ''#{appliance_id}'' is not provided at site with ID '#{service_id}'!" if service_ids.blank?
   end
+
+  def occi_client(endpoint, cert)
+    # TODO: rework client to enable caching
+    @client ||= Occi::Api::Client::ClientHttp.new(endpoint: endpoint,
```

Comment:
It is a workaround for now, client is created separatelly for each contoroller call. Will be reworked in future